python/ray/tune/experiment.py

```diff
@@ -101,6 +121,18 @@ def __init__(
         restore=None,
     ):
 
+        local_dir = _get_local_dir_with_expand_user(local_dir)
+        # `experiment_checkpoint_dir` is for internal use only for better
+        # support of Tuner API.
```

Comment:
shouldn't this go into the doc string for the `Experiment` signature? we should expose document these args, developer API or not.

Reply:
Updated.